test/test_boundary_conditions_integration.jl

```diff
@@ -70,7 +70,7 @@ function fluxes_with_diffusivity_boundary_conditions_are_correct(arch, FT)
     set!(model, b=b₀)
 
     b = model.tracers.b
-    mean_b₀ = mean(interior(b))
```

Comment:
unclear to me what change would make this now fail (scalar indexing error), or why this was written as `mean(interior(` before?

Reply:
`mean` has not always been defined for `Field` and this is old code...

As for an out of bounds error, isn't that a bug?

Reply:
Ah I see, it was a scalar indexing error here which I can reproduce:

```
julia> using Oceananigans
julia> grid = RectilinearGrid(GPU();size=(8, 8, 8), extent = (1, 1, 1))
julia> c = CenterField(grid)
julia> mean(interior(c))
ERROR: Scalar indexing is disallowed.
...
```
but I don't think that's from a bug here its from trying to use the base mean on a CuArray.

Reply:
Its on a view of a CuArray

Reply:
The problem might be related to the fact that `interior(c)` is not a contiguous view into `parent(c)` (because it omits halo regions in x, y). Possibly (but I'm not sure), such `mean` is not supported in CUDA. But I am a little surprised and it could also be a bug / misidrected method issue